### DISCUSSION TITLE
Goldroom黄金屋 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#89] [模板讨论]
Goldroom黄金屋 模板问题反馈评论区
有问题可以下面反馈。

**Comment by xaoei** [comment]
你好,现在签到需要填验证码,是否有方法完成验证?

**Reply by rickychaw** [reply]
使用带ocr的QD镜像

**Comment by rickychaw** [comment]
小反馈 @wjf0214 
![image](https://user-images.githubusercontent.com/18382173/234760212-cefdda15-727f-48fa-a10f-59ba4b2d3319.png)

**Comment by miaoxiaozhao** [comment]
提示【 验证码错误，请重新输入！】，但状态还是【成功】

**Reply by wjf0214** [reply]
根据你的提示，我增加了验证。可以更新下。
